docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx

```diff
@@ -137,7 +146,7 @@ order by 1 desc, 2
 
 :::note
 
-Note that between `2024-02-01` and `2024-06-30` there aren't that many expired persistent entries
+Note that between `2024-02-01` and `2024-10-31` there aren't that many expired persistent entries
```

Comment:
Why does this matter? Curious what we're trying to highlight with the note

Reply:
I was trying to warn that the differences between temp and persistent expiration is pretty large. But yeah that point isn't necessary for this tutorial

Reply:
updated in c560544b37fe5785fb8eac095e6156ecc457b4e0